microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette.md

```diff
@@ -25,15 +25,13 @@ By default, Command Palette isn't enabled. To enable the Command Palette experim
 
 1. Go to `edge://flags`.
 
-<!-- DevTools Toaster is in the process of being changed in Edge to match the feature's name -->
-1. In the **Search flags** text field, type **DevTools Toaster** or **Command Palette**.
+1. In the **Search flags** text field, type **Command Palette**.
```

Comment:
I'm not seeing Command Palette in Stable.  Should the article state that this feature is being rolled out and is appearing in different preview channels of Microsoft Edge in July 2022 timeframe, or state a version #? 

Toaster or Command Palette command is not found in Stable 103.0.1264.62 (Official build) (64-bit).
The C.P. cmd is found in Dev 105.0.1321.0 (Official build) dev (64-bit).

Reply:
pdates the docs in accordance with the new name.

Reply:
@mikehoffms yes, this is only available starting with 105, and the article already mentions this here: https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette?branch=pr-en-us-2071#enable-command-palette